### DISCUSSION TITLE
Pass additional arguments to --entrypoint

### DISCUSSION THREAD

**Discussion by atugushev** [community/community#26979] [Actions]
<p>I have the following workflow:</p>
<pre><code>jobs:
  test:
    runs-on: ununtu-latest
    services:
      nsqd:
        image: nsqio/nsq
        options: --entrypoint /bin/nsqd
        ports:
          - 4150:4150
          - 4151:4151
      nsqlookupd:
        image: nsqio/nsq
        options: --entrypoint /bin/nsqlookupd
        ports:
          - 4160:4160
          - 4161:4161
    env:
      NSQD_HOST: localhost
      NSQLOOKUPD_HOST: localhost
    steps:
      - uses: actions/checkout@master
      - name: Set up Python
        uses: actions/setup-python@v2
        ....
</code></pre>
<p>How to pass an additional argument to the <strong>/bin/nsqd</strong> command in <strong>nsqd</strong> service? For example:</p>
<pre><code>/bin/nsq --lookupd-tcp-address=nsqlookupd:4160
</code></pre>
<p>I’ve tried:</p>
<pre><code>options: --entrypoint="/bin/nsq --lookupd-http-address=nsqlookupd:4161"
</code></pre>
<p>But it raises:</p>
<pre><code>Error response from daemon: OCI runtime create failed: container_linux.go:349: starting container process caused "exec: \"[/bin/nsqd, --lookupd-http-address=nsqlookupd:4161]\": stat [/bin/nsqd, --lookupd-http-address=nsqlookupd:4161]: no such file or directory": unknown
</code></pre>
<p>Google doesn’t help :frowning:</p>

**Comment by mixail-novikov** [comment, marked as answer]
<p>Hello! I’m not an expert just walking around here. But I guess this would help you</p>
<p><a href="https://github.community/t/how-do-i-properly-override-a-service-entrypoint/17435/4">https://github.community/t5/GitHub-Actions/How-do-I-properly-override-a-service-entrypoint/m-p/44258/highlight/true#M5696</a></p>
